libs/islamic-website/index-page/src/lib/funds-block/funds-block.tsx

```diff
@@ -51,7 +51,12 @@ export function FundsBlock() {
               aria-label="Optic Capital"
             >
               <div className="relative h-[80px] w-[100px] opacity-60 transition-opacity duration-300 hover:opacity-100">
-                <Image src={opticCapitalLogoImgData.src} alt="" fill />
+                <Image
+                  src={opticCapitalLogoImgData.src}
+                  alt=""
```

Comment:
set alt for all fund logos like this
```suggestion
                  alt="Optic Capital"
```

Reply:
Done